### PR TITLE
Openvino

### PR DESCRIPTION
**Pull request by wenbinc-Bin**
Add two outputs ssd models support.

### CODE REVIEW

File: libavfilter/dnn/dnn_backend_openvino.c [hunk 1]
```diff
@@ -1267,34 +1254,31 @@ static int get_output_ov(void *model, const char *input_name, int input_width, i
...
-            status = ov_const_port_get_shape(ov_model->input_port, &input_shape);
-            input_shape.dims[2] = input_height;
-            input_shape.dims[3] = input_width;
-            if (status != OK) {
-                av_log(ctx, AV_LOG_ERROR, "Failed create shape for model input resize.\n");
-                return ov2_map_error(status, NULL);
-            }
+        status = ov_partial_shape_create(4, dims, &partial_shape);
+        if (status != OK) {
+            av_log(ctx, AV_LOG_ERROR, "Failed create partial shape.\n");
```
Comment (xhaihao) [inline libavfilter/dnn/dnn_backend_openvino.c]: "Failed to create" ?

File: libavfilter/dnn/dnn_backend_openvino.c [hunk 2]
```diff
@@ -1267,34 +1254,31 @@ static int get_output_ov(void *model, const char *input_name, int input_width, i
...
+            return ov2_map_error(status, NULL);
+        }
+        status = ov_const_port_get_shape(ov_model->input_port, &input_shape);
+        if (status != OK) {
+            av_log(ctx, AV_LOG_ERROR, "Failed create shape for model input resize.\n");
```
Comment (xhaihao) [inline libavfilter/dnn/dnn_backend_openvino.c]: same here and below

File: libavfilter/dnn/dnn_backend_openvino.c (hunk 1 above)
Comment (wenbinc-Bin) [reply libavfilter/dnn/dnn_backend_openvino.c]: Done, Thanks.

File: libavfilter/dnn/dnn_backend_openvino.c (hunk 2 above)
Comment (wenbinc-Bin) [reply libavfilter/dnn/dnn_backend_openvino.c]: Done, Thanks.

**Review by xhaihao — APPROVED** [review]
LGTM

**Comment by wenbinc-Bin** [comment]
https://patchwork.ffmpeg.org/project/ffmpeg/list/?series=10363

**Pull request closed without merging**